### DISCUSSION TITLE
Extend JSON Responses

### DISCUSSION THREAD

**Discussion by andyrue** [janko/rodauth-rails#71] [Q&A] [answered]
I'm trying to figure out if there is a way to include the `current_user` in the JSON response after a successful login instead of just the `success` notice message. I can make an additional call after logging in, but it would be nice to get it in one response. Any recommendations?

**Comment by renchap** [comment, marked as answer]
Yes, you can use `json_response` for this.

For example:

```ruby
after_login do
      json_response.merge!(user: user.as_json)
end
```

**Comment by andyrue** [comment]
Excellent. Thank you!
